package.json

```diff
@@ -84,12 +84,12 @@
     "typescript": "^4.8.4"
   },
   "peerDependencies": {
-    "@expo/config-plugins": "^4.1.0",
+    "expo": ">=47.0.0",
```

Comment:
this should continue to work indefinitely into the future, and only break if there is a breaking change to the config plugins APIs that we depend on here.